plugin.video.vtm.go/addon.xml

```diff
@@ -22,8 +22,9 @@
         <disclaimer lang="nl_NL">Deze add-on wordt niet ondersteund door DPG Media en wordt aangeboden 'as is', zonder enige garantie. De VTM GO naam, VTM GO logo en de kanaallogo's zijn eigendom van DPG Media en worden gebruikt in overeenstemming met de fair use policy.</disclaimer>
         <platform>all</platform>
         <license>GPL-3.0</license>
-        <news>v1.0.2
-- Fixes authentication errors</news>
+        <news>v1.1.0
```

Comment:
There are much more changes in the add-on than just these two. Why not give the most important ones?

Reply:
Most of the merge requests are for IPTV Manager support. One improves subtitles, and another fixes a potential issue when tokens would be invalidated (but that hasn't happened up till now). The rest are for improved testing (but these aren't in this PR).

So besides IPTV Manager support, and improved subtitles, there isn't much other stuff for the users.